pkgs/applications/networking/instant-messengers/profanity/default.nix

```diff
@@ -8,25 +8,27 @@
 , traySupport ? true,           gnome2 ? null
 , pgpSupport ? true,            gpgme ? null
 , pythonPluginSupport ? true,   python ? null
+, omemoSupport ? true,          libsignal-protocol-c ? null, libgcrypt ? null
 }:
 
 assert autoAwaySupport     -> libXScrnSaver != null && libX11 != null;
 assert notifySupport       -> libnotify != null && gdk_pixbuf != null;
 assert traySupport         -> gnome2 != null;
 assert pgpSupport          -> gpgme != null;
 assert pythonPluginSupport -> python != null;
+assert omemoSupport        -> libsignal-protocol-c != null && libgcrypt != null;
 
 with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "profanity-${version}";
-  version = "0.6.0";
+  version = "0.7.0";
 
   src = fetchFromGitHub {
-    owner = "boothj5";
+    owner = "profanity-im";
```

Comment:
Please explain why this change can be trusted, preferably citing an official source.

Reply:
I've just noticed that the link https://github.com/boothj5/profanity redirects to https://github.com/profanity-im/profanity, so I'm assumed that it's better to use this. I'm a new user of profanity since it has OMEMO-support, and I cannot find an official resource announcing that change. If the redirect is not enough to trust this change I will ask on mailing list for a official statement.

Reply:
@toon23 No need, that is sufficient. I just like to encourage people to provide that information in their PRs. Thanks :smile: